docs/docs/user/references/editor-integration.md

```diff
@@ -17,9 +17,9 @@ KPOps provides JSON schemas that enable autocompletion and validation for some o
 ??? note "`settings.json`"
 
     ```json
-      --8<--
-      ./docs/resources/editor_integration/settings.json
-      --8<--
+    --8<--
+    ./docs/resources/editor_integration/settings.json
+    --8<--
```

Comment:
is this the correct indentation now?

Reply:
do you want to reference the issue in this PR or is there further work required to close it?